src/main/java/hardcorelife/chryscorelab/Touchy.java

```diff
@@ -196,4 +199,20 @@ public boolean naturalRegEnabled() {
         // Returns the value of natural_regeneration from config.yml
         return getConfig().getBoolean("natural_regeneration", false);
     }
-}
+
+    public boolean killGainLivesEnabled() {
+        // Returns the value of kill_gain_lives from config.yml
+        if (globalLivesEnabled()){
+            return false;
```

Comment:
Any thoughts on allowing `killGainLivesEnabled` to be toggleable with `globalLivesEnabled`? (see [PlayerDeath.java](https://github.com/Chryscorelab/HardcoreLife/blob/9aca82450912502f968e2fe7dc7e0adb9ce426ed/src/main/java/hardcorelife/chryscorelab/listeners/PlayerDeath.java))

Since we're both adding lives and checking for permadeath when receiving a player death event, we could check to see if a life should be granted before accounting for the life removal to prevent a permadeath event when there are 1 lives remaining. 

For example, we could move the [current check](https://github.com/Chryscorelab/HardcoreLife/blob/9aca82450912502f968e2fe7dc7e0adb9ce426ed/src/main/java/hardcorelife/chryscorelab/listeners/PlayerDeath.java#L81) to inside the else statement that handled when global_lives is false, then add a check for when global_lives is true to ignore that death event. We could also publish a message saying that no lives were removed or something.

What do you think?

Reply:
Indeed all this can be a good idea!

Reply:
Just added in the new push (I hope that it corresponds to the expectations)